src/prefect/cli/work_pool.py

```diff
@@ -771,14 +775,11 @@ async def storage_inspect(
 
 @work_pool_storage_configure_app.command()
 async def s3(
-    work_pool_name: Annotated[
-        str,
-        typer.Argument(
-            ...,
-            help="The name of the work pool to configure storage for.",
-            show_default=False,
-        ),
-    ],
+    work_pool_name: str = typer.Argument(
+        ...,
```

Comment:
not a big thing but should we accept this only as a positional argument or allow a `--work-pool-name` flag too? im just thinking that 2 positional arguments after 3 more "positional" subcommands is pretty hard to read imo

Reply:
Yeah, these commands are quite long, but I think it makes sense to keep the work pool name as a positional argument to be consistent with the rest of the work pool commands and because adding a flag would require even more keystrokes. I'm open to ideas for how to shorten these commands.